projects/ui/src/components/Nav/FolderMenu.tsx

```diff
@@ -84,6 +101,25 @@ const FolderMenu: FC<{
     close();
   };
 
+  useEffect(() => {
+    function handleResize() {
+      if (window.innerWidth < theme.breakpoints.values.lg && !mobileWindow) {
+        setMobileWindow(true);
+      } else {
+        setMobileWindow(false);
+      }
+    }
+    window.addEventListener('resize', handleResize);     
+    return () => window.removeEventListener('resize', handleResize);
+  }, []);
```

Comment:
Future work: Switch this to the `useMediaQuery` API which optimizes event listeners. This implementation works but will be slow, especially when the user is dragging their screen size to be smaller.

https://mui.com/material-ui/react-use-media-query/

Alternatively, the JS-based style strategy could be replaced with a CSS-based one where a new component is rendered after a certain breakpoint.

Reply:
Tracking here: #438